style.css

```diff
@@ -0,0 +1,806 @@
+  padding: 4rem 0;
+  background: #000;
+  flex-direction: row;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  justify-items: center;
+  clear: both;
+}
+
+.footer-links {
+  flex-direction: row;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  justify-items: center;
+  background: #000;
+  color: white;
+}
+
+.footer-links a {
+  padding-left: 20px;
+  margin: 10px;
+  color: white;
+}
+
+.copyright {
+  display: flex;
+  justify-content: center;
+  padding: 4rem 0;
+  font-size: 1.3rem;
+  color: #999;
+  background: #000;
+  text-align: center;
+}
+
+.copyright span {
+  color: #f42;
+}
+
+.img-art2 {
+  width: 100%;
+  height: 153px;
+  background-image: url('https://cdnp3.stackassets.com/f54c9185d68a19989ac1618bb5f955852e40ea0e/store/988cf18b55dc05a3eca624beef4de98a661b6f75463223a0c57759f988b1/sale_23318_primary_image.jpg');
+  background-position: center center;
+  background-size: 100% 100%;
+}
+
+.deals {
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  display: grid;
+  grid-gap: 20px;
+  margin-bottom: 20px;
+}
+
+.news {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 20px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.funding-rounds {
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  display: grid;
+  grid-column-gap: 20px;
+  line-height: 3;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+p.asset-text {
+  padding-left: 100px;
+}
+
+.funding-rounds p.asset-text {
+  line-height: 1;
+  margin-top: 0;
+  margin-bottom: 0;
+  font-size: 14px;
+  padding-left: 100px;
+}
+
+.asset-text1 {
+  background-color: #f3cdcd;
+  margin-top: 0;
+  line-height: 1;
+  padding-bottom: 10px;
+  padding-top: -5px;
+}
+
+.funding-rounds img {
+  width: 70px;
+  float: left;
+  margin-top: 9px;
+  margin-left: 12px;
+}
+
+.clear-text {
+  clear: both;
+  padding-left: 26px;
+}
+
+.asset-text {
+  background-color: #f3cdcd;
+  margin-top: 0;
+  padding-top: 12px;
+}
+
+.company-name {
+  margin-top: 0;
+  padding-bottom: 22px;
+  padding-top: 8px;
+  margin-bottom: 0;
+  background-color: #ff642d;
+}
+
+.funding-rounds p {
+  padding-left: 30px;
+  border-top-color: #000;
+}
+
+.funding-rounds div {
+  background-color: #eee0e0;
+}
+
+p.asset-text1 {
+  padding-left: 100px;
+}
+
+p.company-name {
+  padding-left: 100px;
+}
+
+.main-art {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 10px;
+  color: #fff;
+  width: 100%;
+  margin-top: 89px;
+  height: 75vh;
+  max-height: 560px;
+}
+
+.latest img {
+  width: 120px;
+  margin-top: -25px;
+  float: right;
+  height: 96px;
+}
+
+.articles {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 20px;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.articles h2 {
+  color: #ff7900;
+  font-size: 21px;
+  margin-left: 5px;
+}
+
+.articles h3 {
+  color: white;
+  font-size: 15px;
+  margin-left: 5px;
+}
+
+.articles h5 {
+  color: grey;
+  font-size: 10px;
+  margin-left: 5px;
+}
+
+.corona-track p {
```

Comment:
- [ ] Remember to add padding to the sides of the text too. It's too close to the border of the image like the previous TSE pointed out. 
- [ ] [OPTIONAL] Flex would be better than a large padding like this to align the element to the bottom of its parent.